src/features/map/components/Map/Map.css

```
@@ -15,16 +15,14 @@
 
-.layer-toggle {
-  padding: 5px 12px;
-}
-
 .layer-toggle .layer-toggle-label {
   display: flex;
   align-items: center;
   gap: 8px;
+  padding: 5px 12px;
 
   user-select: none;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)